.eslintrc.js

```diff
@@ -8,6 +8,7 @@ const config = {
     'plugin:@typescript-eslint/recommended-requiring-type-checking',
     'plugin:react/recommended',
     'plugin:react-hooks/recommended',
+    'plugin:tailwindcss/recommended',
```

Comment:
i've only used the prettier-plugin before - how does this compare?

I was thinking of making a PR just now migrating to tailwind 3.3 with TS config, does this support that? We added support to the prettier plugin here: https://github.com/tailwindlabs/prettier-plugin-tailwindcss/pull/137

Reply:
This eslint plugin is similar to the prettier one (to order class names) but also adds some [interesting rules](https://github.com/francoismassart/eslint-plugin-tailwindcss#supported-rules).

`tailwind.config.ts` doesn't seem to be supported yet, but I'll happily dig into adding support for it: https://github.com/francoismassart/eslint-plugin-tailwindcss/issues/228